packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -58,7 +58,24 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 				allConversations,
 				allSupportInteractions
 			);
-			setConversations( filteredConversations );
+
+			const conversationsWithUpdatedStatuses = filteredConversations.map( ( conversation ) => {
+				const supportInteraction = allSupportInteractions.find(
+					( interaction ) => interaction.uuid === conversation.metadata?.supportInteractionId
+				);
+
+				const updatedConversation = {
+					...conversation,
+					metadata: {
+						...conversation.metadata,
+						...( supportInteraction?.status && { status: supportInteraction.status } ),
```

Comment:
If we don't have a matching `supportInteraction`, should we even modify `conversation` in the first place? We could return early in that case.

Reply:
Yup good catch.

Reply:
Thanks for addressing. But that also means we can remove the optional chaining here too?

Also, if `status` is always there, perhaps we can remove the `supportInteraction?.status &&` check altogether?